### PR TITLE
[TC-252] UI bugs

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
[TC-252](https://bcdevex.atlassian.net/browse/TC-252)

Objective:

- UI bugs
- remove unused image logos

[TC-252]: https://bcdevex.atlassian.net/browse/TC-252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: frontend/src/components/table/classes.ts [hunk 1]
```diff
@@ -42,10 +42,10 @@ export const tableClass = (key: string, value?: string) => {
         : tableClasses.unavailable;
     case DashboardColumns.TRAVEL:
       return value === 'yes' ?  tableClasses.travelYes : tableClasses.travelNo;
-    case DashboardColumns.CLASSIFICATION:
-      return value === ClassificationName.EXCLUDED
+    case DashboardColumns.UNION_MEMBERSHIP:
+      return value === Classification.EXCLUDED
         ? tableClasses.excluded
-        : tableClasses.bcgeu;
+        : value === Classification.BCGEU ? tableClasses.bcgeu : tdClass;
```
Comment (fw-noel) [inline frontend/src/components/table/classes.ts:48]: This is a bit strange. What's the requirement here?

Comment (chelsea-EYDS) [reply frontend/src/components/table/classes.ts:48]: The color is different depending on the value

Comment (fw-noel) [reply frontend/src/components/table/classes.ts:48]: Sorry, I get that, so it's three different colors? `excluded`, `bcgeu`, and neither?

Comment (chelsea-EYDS) [reply frontend/src/components/table/classes.ts:48]: Going to make a change on this to match the new design req's (I spoke with Chevonne earlier and there will now be a color for each value)

**Review by fw-noel — APPROVED** [review]

**Pull request merged**